services/ui-src/src/utils/auth/UserProvider.tsx

```diff
@@ -74,6 +74,8 @@ export const UserProvider = ({ children }: Props) => {
       const userRole = cms_role.split(",").find((r) => r.includes("mdctmfp"));
       const state = payload["custom:cms_state"] as string | undefined;
       const full_name = [given_name, " ", family_name].join("");
+      const reports = payload["custom:reports"] as string | undefined;
+      const userReports = !reports ? ["WP", "SAR"] : reports.split(",");
```

Comment:
Note: Can delete this stuff because we're not restricting any users from SAR or WP.